dot/state/epoch_test.go

```diff
@@ -591,12 +597,24 @@ func TestStoreAndFinalizeBabeNextConfigData(t *testing.T) {
 			err := epochState.blockState.db.Put(headerKey(tt.finalizeHash), []byte{})
 			require.NoError(t, err)
 
+			// before finalize next config data this epoch should not be defined
+			definedEpochData, definedConfigData, err := epochState.AlreadyDefined(tt.finalizeEpoch)
+			require.NoError(t, err)
+			require.False(t, definedEpochData)
+			require.False(t, definedConfigData)
+
 			err = epochState.FinalizeBABENextConfigData(tt.finalizeEpoch)
 			if tt.expectErr != nil {
 				require.ErrorIs(t, err, tt.expectErr)
 			} else {
 				require.NoError(t, err)
 
+				// after finalize next config data this epoch should be defined
```

Comment:
nit maybe a word missing I think?

```suggestion
				// after finalize next config data for this epoch should be defined
```